site/pages/docs/ref/add-cal/add-cal-en.hbs

```diff
@@ -0,0 +1,107 @@
+---
+{
+	"title": "Add to calendar",
+	"language": "en",
+	"category": "Plugins",
+	"categoryfile": "plugins",
+	"description": "Add an event to a calendar (google, Apple, Android, Outlook and others).",
+	"altLangPrefix": "add-cal",
+	"dateModified": "2021-05-12"
+}
+---
+<h2>Purpose</h2>
+<p>Add an event to a calendar (google, Apple, Android, Outlook and others).</p>
+
+<h2>Working example</h2>
+<ul>
+	<li><a href="../../../demos/add-cal/add-cal-en.html">English example</a></li>
+	<li><a href="../../../demos/add-cal/add-cal-fr.html">French example</a></li>
+</ul>
+
+<h2>Evaluation and report</h2>
+<p>There is no evaluation and report available for this component.</p>
+
+<h2>API (Version 1.0)</h2>
+<dl class="dl-horizontal">
+	<dt>Function</dt>
+	<dd>provisional</dd>
+	<dt>Configuration</dt>
+	<dd>Not applicable</dd>
+	<dt>User interface (Template)</dt>
+	<dd>provisional</dd>
+	<dt>Data source</dt>
+	<dd>provisional</dd>
+	<dt>View and style</dt>
+	<dd>provisional</dd>
+	<dt>i18n string</dt>
+	<dd>provisional</dd>
+</dl>
+
+<h3>Function</h3>
+<p>(Version 1.0)</p>
+<table class="table">
+	<tr>
+		<th>Function type</th>
+		<th>Name</th>
+		<th>How to implement</th>
+		<th>What it does</th>
+	</tr>
+	<tr>
+		<td>jQuery Event</td>
+		<td><code>wb-init.wb-addcal</code></td>
+		<td>Triggered manually (e.g., <code>$( ".wb-addcal" ).trigger( "wb-init.wb-addcal" );</code>).</td>
+		<td>Initializes the bg-img plugin. This plugin will be initialized automatically unless the <code>.wb-wb-addcal</code> element is added after the page load and wet-boew was initialized.</td>
+	</tr>
+	<tr>
+		<td>jQuery Event</td>
+		<td><code>wb-ready.wb-addcal</code></td>
+		<td>Triggered automatically after the plugin initializes.</td>
+		<td>Used to identify when and where the plugin initializes (target of the event).
+			<pre><code>$( document ).on( "wb-ready.wb-addcal", ".wb-addcal", function( event ) {
+});</code></pre>
+			<pre><code>$elm.on( "wb-ready.wb-addcal", function( event ) {
+});</code></pre>
+		</td>
+	</tr>
+</table>
+
+<h3>Configuration</h3>
+<p>Not applicable for this plugin.</p>
+
+<h3>User interface (Template)</h3>
+<p>(Provisional)</p>
+
+<p>The Schema.org <code>Event</code> type noted in RDFa is required when implementing this add to calendar widget. It identifies an event happeing at a certain time and location. The followings are Schema.org types and properties:</p>
+<dl>
+	<dt><code>name</code> (Event)</dt>
+	<dd>Required - Short text that defines the event title.</dd>
+	<dt><code>startDate</code> (Event)</dt>
+	<dd>Required - Specifies the start date and or time of the event. The following pattern must be used: "YYY-MM-DD HH:MMZ"</dd>
```

Comment:
Please add this need to be used with a `<time>` element and the value must be defined through the `datetime` attribute.

Idem for `endDate`